adafruit_trellism4.py

```diff
@@ -171,6 +176,12 @@ class TrellisM4Express:
     """
     def __init__(self, rotation=0):
         self._rotation = rotation
+
+        # Define accelerometer
+        self._i2c = busio.I2C(board.ACCELEROMETER_SCL, board.ACCELEROMETER_SDA)
+        self._adxl345 = adafruit_adxl34x.ADXL345(self._i2c)
```

Comment:
`self._i2c` is not used elsewhere, so you could just make it a local variable:
```
i2c = busio.I2C(board.ACCELEROMETER_SCL, board.ACCELEROMETER_SDA)
self._adxl345 = adafruit_adxl34x.ADXL345(i2c)
```
or even
```
self._adxl345 = adafruit_adxl34x.ADXL345(busio.I2C(board.ACCELEROMETER_SCL, board.ACCELEROMETER_SDA))
```

Reply:
I set this up the same way it's setup in the CPX lib. But I can change it here.

Reply:
It saves a little space not to have the I2C object stored in the `TrellisM4_Express` object.. It's already stored in the `ADXL345` object too, so if you wanted it, you could get it from there.